src/styles/blocks/star.css

```diff
@@ -0,0 +1,16 @@
+.stars {
```

Comment:
The block name is `stars`, so probably the file should also be named `stars.css`

Reply:
done